torch/_tensor.py

```diff
@@ -394,25 +394,36 @@ def _reduce_ex_internal(self, proto):
             )
             return (torch._utils._rebuild_wrapper_subclass, arg_wrapper_subclass)
         else:
-            # TODO: Once we decide to break serialization FC, no longer
-            # need to wrap with TypedStorage
-            args = (
-                torch.storage.TypedStorage(
+            v3_dtypes = [torch.float8_e5m2, torch.float8_e4m3fn]
```

Comment:
also `uz` dtypes?

Reply:
Probably makes sense to serialize all tensors whose dtype is not in the dict like this.

Reply:
Hmm, so do the inverse. Makes sense, will do in follow up PR (and probably past the branch cut, as it feels a bit risky)